DeepCat.js

```diff
@@ -139,14 +139,14 @@
 	}
 
 	function getAjaxRequest( searchTerm, searchTermNum ) {
-		var categoryString = DeepCat.extractDeepCatCategory( searchTerm ),
-			userParameter = {
+		({  categoryString, depth } = DeepCat.extractDeepCatCategory( searchTerm ));
```

Comment:
I find it hard to. But I always wonder if it is actually hard or if I'm just not used to it yet…